scripts/setup.sh

```diff
@@ -18,10 +20,14 @@ function setup_build_tools() {
         cd ${topdir}/tools/
 
         echo "> Arm Toolchain: not found. Downloading .." 
-        wget https://developer.arm.com/-/media/Files/downloads/gnu-a/9.2-2019.12/binrel/gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz
-        tar -Jxf gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz
-        rm gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz
-        echo "> Arm Toolchain: downloaded .." 
+        wget https://developer.arm.com/-/media/Files/downloads/gnu-a/9.2-2019.12/binrel/gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz &>>/dev/null
+		if [ $? -eq 0 ]; then
+			echo "> Arm Toolchain: downloaded .."
+			tar -Jxf gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz &>>"${LOG_FILE}"
+			rm gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf.tar.xz
+		else
+			echo "> Arm Toolchain: could not download"
```

Comment:
Better say Failed to download along with the return code to give a more clear perspective of what happened.